src/app/navbar/navbar.component.css

```diff
@@ -0,0 +1,42 @@
+* {
+    font-family: Roboto;
+}
+
+.menu-container {
+    padding-bottom: 25px;
+    display: flex;
+    padding-left: 115px;
+    background-color: #003c4d;
+    height: 122px;
+}
+
+.menu-wrapper {
+    padding-top: 18px;
+}
+
+.menu-link {
+    font-size: 12px;
+    color: #ffffff;
+    letter-spacing: 0.36px;
+    line-height: 1.2rem;
```

Comment:
Question: In most cases, you are using pixel values, but here you are using rem. Why?